doc/pages/architecture/090-database.md

```diff
@@ -0,0 +1,219 @@
+# Astarte Database
+
+Astarte might need to store a massive ammount of data, to accomplish this task a [Cassandra](http://cassandra.apache.org/) database is used. Cassandra offers scalability and high availability with [good performances](https://www.datastax.com/apache-cassandra-leads-nosql-benchmark).
+Cassandra offers linear scalability and can span from really small clusters to thousands of nodes, last but not least it provides a reliable storage for all devices data.
+[ScyllaDB](https://www.scylladb.com/) >= 2.5 is also supported as a drop-in replacement when a [performances boost](https://www.scylladb.com/product/benchmarks/) is needed.
+
+Cassandra is also the ideal storage for large-scale data processing with [Apache Spark](http://spark.apache.org/).
+
+Astarte is multi-tenant by design, each tenant has its own realm. Each realm has its own Cassandra keyspace, that can be tuned according to realm specific needs (e.g. each realm might have a different replication level). For this reason here realm and keyspace can be used as synonyms, except for the `astarte` keyspace.
+
+## Schema and Keyspace Creation
+
+Astarte automatically takes care of keyspaces and tables creation (those tasks are performed by `astarte_housekeeping`), here are documented for completeness.
```

Comment:
Astarte automatically takes care of keyspaces, tables creation and intra-version migrations (those tasks are performed by `astarte_housekeeping` or `astarte_realm_management`, depending on the context). The following documentation is just a reference about Astarte's internal statements, and is related to the release series referenced by the documentation.